composer.json

```
@@ -22,8 +22,8 @@
     "require-dev": {
         "phpro/grumphp": "^0.17.1 || ^0.19.0",
         "phpstan/phpstan": "^0.11.15 || ^0.12.0",
-        "phpunit/phpunit": ">=7.5",
-        "squizlabs/php_codesniffer": "^3.0",
+        "phpunit/phpunit": ">=9.0",
```

```suggestion
        "phpunit/phpunit": "^8.0 || ^9.0",
```
This will maintain compatibility with antidot-fw/framework package